src/lib.rs

```diff
@@ -737,4 +820,32 @@ mod tests {
         assert_eq!(hash.get("foo"), Some(&42));
         assert_eq!(hash.get("bar"), Some(&23));
     }
+
+    #[test]
+    fn owning_handle() {
+        use std::cell::RefCell;
+        let cell = Rc::new(RefCell::new(2));
+        let cell_ref = RcRef::new(cell);
+        let mut handle = OwningHandle::new(cell_ref, |x| unsafe { x.as_ref() }.unwrap().borrow_mut());
```

Comment:
IIUC it's an unbounded lifetime: https://doc.rust-lang.org/nomicon/unbounded-lifetimes.html